Packages/com.unity.inputsystem/InputSystem/Editor/DeviceSimulator/InputSytemPlugin.cs

```diff
@@ -0,0 +1,92 @@
+#if UNITY_EDITOR && UNITY_2021_1_OR_NEWER
+
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEditor.DeviceSimulation;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
+
+public class InputSystemPlugin : DeviceSimulatorPlugin
+{
+    private bool m_InputSystemEnabled;
+    private Touchscreen m_SimulatorTouchscreen;
+    private List<InputDevice> m_DisabledDevices;
+
+    public override string title => "Input System";
+
+    public override void OnCreate()
+    {
+        var playerSettings = Resources.FindObjectsOfTypeAll<PlayerSettings>()[0];
+        SerializedObject playerSettingsSerialized = new SerializedObject(playerSettings);
+
+        var activeInputHandler = playerSettingsSerialized.FindProperty("activeInputHandler").intValue;
+        m_InputSystemEnabled = activeInputHandler == 1 || activeInputHandler == 2;
+        if (m_InputSystemEnabled)
+        {
+            deviceSimulator.touchScreenInput += OnTouchEvent;
+
+            // UGUI elements like a button don't get pressed when multiple pointers for example mouse and touchscreen are sending data at the same time
+            m_DisabledDevices = new List<InputDevice>();
+            foreach (var device in InputSystem.devices)
```

Comment:
This will not work correctly with devices that are added and removed on the fly.

Reply:
Added code to handle it during InputSystem.onDeviceChange. Was not actually able to make it fire, all devices in the editor seem to be pre-added and disconnecting reconnecting does not seem to do anything...